packages/byods-demo-server/babel.config.js

```diff
@@ -0,0 +1,3 @@
+const babelConfigLegacy = require('@webex/babel-config-legacy');
```

Comment:
This was a part of the code cleanup - better to have .js file rather than a json one.